lib/PuppeteerSharp/Frame.cs

```diff
@@ -98,6 +99,29 @@ internal Frame(FrameManager frameManager, CDPSession client, Frame parentFrame,
 
         #region Public Methods
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="url"></param>
+        /// <param name="options"></param>
+        /// <returns></returns>
+        public async Task<Response> GoToAsync(string url, NavigationOptions options)
+        {
+

```

Comment:
This can be an arrow function.
```cs
public Task<Response> GoToAsync(string url, NavigationOptions options)
   => FrameManager.Navigate(this, url, options);
```